src/index.js

```diff
@@ -554,7 +554,17 @@ class EliteDangerousJournalServer {
     if (event === 'change' && filepath === this.currentJournal) {
       this.getJournalUpdate();
     } else {
-      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occured for`)} ${chalk.magenta(filepath)}`);
+      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occurred for`)} ${chalk.magenta(filepath)}`);
```

Comment:
I don't know if we need the `console.log()` here any more. I thin it was mostly for debugging in case anything else triggered a file change.

Reply:
agreed.